mne_bids/mne_bids.py

```diff
@@ -708,7 +708,8 @@ def write_raw_bids(raw, bids_basename, output_path, events_data=None,
                     'Please contact MNE developers if you have '
                     'any questions.')
         else:
-            raw.save(bids_fname, overwrite=True, split_naming='bids')
+            # This ensures that single FIF files do not have the part param
+            raw.save(bids_fname, overwrite=True, split_naming='neuromag')
```

Comment:
can you add a test to check this?

Reply:
hmm...does the validator allow files that have `_part-01` in the name but no subsequent parts? to add a test here, it would essentially be a copy of the mne-python split files testing, should I just do that?

Reply:
The validator unfortunately does not throw an error here

But you could just check the filename in the folder it's written to ...